pkgs/servers/monitoring/nagios-plugins/openbsd_snmp3_check/default.nix

```
@@ -0,0 +1,48 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  openbsd_snmp3_check,
+  python3,
+  stdenv,
+  testers,
+}:
+stdenv.mkDerivation rec {
+  pname = "openbsd_snmp3_check";
+  version = "0.55";
+
+  src = fetchFromGitHub {
+    owner = "alexander-naumov";
+    repo = "openbsd_snmp3_check";
+    rev = "v${version}";
+    hash = "sha256-qDYANMvQU72f9wz8os7S1PfBH08AAqhtWLHVuSmkub4=";
+  };
+
+  buildInputs = [
+    python3
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir --parents $out/bin
+    install --mode=0755 openbsd_snmp3.py $out/bin
+    runHook postInstall
+  '';
```

What do you think about simplifying the installPhase into this?

```suggestion
  postInstall = ''
    install -Dm555 openbsd_snmp3.py $out/bin/openbsd_snmp3.py
  '';
```

---

I think that's better. Is there a preference on short options vs long options?